other-examples/collector/gcp-cloud-spanner/README.md

```diff
@@ -0,0 +1,64 @@
+# GCP Cloud Spanner OpenTelemetry metrics example setup
+
+This example shows a setup for running a Docker OpenTelemetry Collector to scrape metrics from your GCP Cloud Spanner instance and post them the New Relic OTLP Collector Endpoint. You can view the metrics out of the box in New Relic by adding the [GCP Cloud Spanner dashboard](https://newrelic.com/instant-observability/google-cloud-spanner-otel) to your account.
```

Comment:
Let's add a TODO here to update the dashboard to reflect the shape of the data being sent up, and update the README to include details on how to install the dashboard when available.